num_func/src/lib.rs

```diff
@@ -1,28 +0,0 @@
-use gandiva_rust_udf_macro::udf;
-
-#[udf]
-fn gcd(x: i64, y: i64) -> i64 {
```

Comment:
num related functions are still useful, and we could keep `gcd` and `lcm` for production use